libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -773,11 +813,13 @@ export function Tracker(
           updateFirstEventInIdCookie(idCookie, payloadBuilder);
           incrementEventIndexInIdCookie(idCookie);
 
-          payloadBuilder.add('vp', detectViewport());
-          payloadBuilder.add('ds', detectDocumentSize());
+          const { viewport, documentSize } = getBrowserProperties();
+
+          payloadBuilder.add('vp', viewport);
```

Comment:
Question: there was a discussion whether we should still set the canonical event properties when the browser context is enabled in v4 – where do we land on that? It's probably a big change to stop tracking them by default, but maybe we could let users optionally disable them?

Reply:
Based on an offline discussion we can make this context an opt-out and remove these properties. However let's discuss it more in the context of v4.